docs/LwM2M_Client/OpenThread/Adding_device.md

```diff
@@ -0,0 +1,145 @@
+# Adding nRF board to OpenThread
+
+Integrate your Nordic nRF52840-DK board to an active OpenThread Border Router.
+
+## Prerequisites
+
+- An active **OpenThread Border Router**.
+- The **nRF52840-DK** board with a USB cable.
+- Installed **minicom** or **RealTerm** (for Linux or Mac) or **PuTTy** (for Windows) or other serial communication program.
+- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+- A user with access to the [**Coiote IoT Device Management**](https://eu.iot.avsystem.cloud/ui/device/inventory) platform and appropriate permissions.
+
+## Connecting to the OpenThread
+
+Check your active Border Router **OTBR_IP_address** and **OTBR_port** on which your Border Router works. This will be necessary for further steps.
+
+### Connection through Wi-Fi
+
+You have to be connected to the same Wifi network as your OpenThread Border Router. To connect your device to Border Router through Wifi, you should go to `OTBR_IP_address`.
```

Comment:
It's not clear what you mean by "go to `OTBR_IP_address`. Maybe state that you can connect to the WebUI using a web browser by connecting to `OTBR_IP_address:OTBR_port`